ffi/capi/src/decimal.rs

```diff
@@ -92,14 +95,15 @@ pub mod ffi {
                 *new = char::from_u32(old).unwrap_or(char::REPLACEMENT_CHARACTER);
             }
             let digits = new_digits;
-            let plus_sign_affixes = AffixesV1 {
-                prefix: str_to_cow(plus_sign_prefix),
-                suffix: str_to_cow(plus_sign_suffix),
-            };
-            let minus_sign_affixes = AffixesV1 {
-                prefix: str_to_cow(minus_sign_prefix),
-                suffix: str_to_cow(minus_sign_suffix),
+            let strings = DecimalSymbolsV1Strings {
+                plus_sign_prefix: str_to_cow(plus_sign_prefix),
+                plus_sign_suffix: str_to_cow(plus_sign_suffix),
+                minus_sign_prefix: str_to_cow(minus_sign_prefix),
+                minus_sign_suffix: str_to_cow(minus_sign_suffix),
+                decimal_separator: str_to_cow(decimal_separator),
+                grouping_separator: str_to_cow(grouping_separator),
             };
```

Comment:
Suggestion: write this in a way that avoids allocating 6 strings and then the MulfiFieldsULE box. I think you should be able to make these be borrowed in the `DecimalSymbolsV1Strings`.

Reply:
^ do you plan to do this?